style.css

```diff
@@ -133,343 +146,228 @@ section {
+  background-color: #396df2;
 }
 
-.section-two-2 .background .box div a {
-  margin-right: 15px;
-  background: #ebebff;
-  border-radius: 8px;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 4px 12px;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 12px;
-  line-height: 16px;
-  color: #6070ff;
-  margin-top: 15px;
-  text-decoration: none;
-}
-
-.section-two-2 .background button {
-  margin-top: 20px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 7px;
-  border: 2px solid #dfe1e6;
+.case1 .case2 button {
+  border: 1px solid #6070ff;
+  padding: 10px;
   border-radius: 10px;
-  gap: 12px;
-  background: #fff;
+  margin-top: 15px;
+  background-color: #f7f7f9;
+  transition: all 1s;
 }
 
-/* the about area */
-.about-myself-container {
-  width: 375px;
-  height: 850px;
-  left: 375px;
-  top: 4769px;
-  background: #fff;
+.case1 .case2 button:hover {
+  background-color: #396df2;
 }
 
-.about-myself-container .dropdown-menu {
-  display: grid;
-  row-gap: 20px;
-  margin-top: 40px;
+/* The about section area */
+.about-section {
+  padding: 100px 20px 0 20px;
 }
 
-.about-myself-container .dropdown-menu p {
-  font-style: normal;
+.about-section h3 {
   font-weight: 500;
-  font-size: 20px;
+  font-size: 16px;
   line-height: 24px;
+  color: #7f8cff;
+  margin-bottom: 20px;
 }
 
-.about-myself-container .dropdown-menu .drop1 ul li a {
+.about-section .icon-two ul {
   display: flex;
-  align-items: center;
-  padding-right: 20px;
+  margin-bottom: 20px;
 }
 
-.form-box .text ul li {
+.about-section .icon-two ul li {
   list-style-type: none;
+  margin-right: 20px;
 }
 
-.about-myself-container .dropdown-menu .drop1 ul li {
-  list-style-type: none;
-  width: 303px;
-  height: 66.72px;
-  background: #f7f7f9;
-  margin-left: 20px;
+.about-section div ul ul li img {
+  margin-right: 20px;
 }
 
-.form-box .text ul {
-  display: flex;
-  justify-content: center;
+.about-section div ul .link2,
+.link3 {
+  border-bottom: 1px solid #dfe1e6;
 }
 
-.about-myself-container .dropdown-menu .drop1 ul {
-  margin-left: -5px;
+.body-section {
+  margin-bottom: 2100px;
 }
 
-/* form area */
-.about-myself-container .dropdown-menu .drop2 p {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  border-bottom: 1px solid #dfe1e6;
-  width: 303px;
-  height: 66.72px;
-  padding-top: 20px;
-}
+/* contact/form area */
 
-.form-box {
+.contact-container {
+  padding: 100px 0 30px 0;
   background-image: url(image/Contact-me.png);
-  width: 303px;
-  height: 700px;
-  top: 5410px;
-  margin-top: 100px;
-  margin-left: 20px;
 }
 
-.form-box .text ul:nth-child(1) li {
-  font-style: normal;
+.contact-container .contact-me {
+  display: flex;
+  justify-content: center;
   font-weight: 700;
-  font-size: 40px;
+  font-size: 25px;
   line-height: 52px;
-  margin-top: 80px;
   color: #fff;
+  margin-left: 20px;
+  margin-right: 20px;
 }
```

Comment:
- [ ] Kindly make sure you stick to the template as much as possible e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200566637-cb0066f9-2a83-4b60-b580-73ac2caa3699.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200567529-d218c52e-0eec-4c3e-a42f-7f8e449b07e4.png)